resources/deploy_intermediate_debian.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
+
+#
+# Deploy an intermediate webserver and set up automatic VPN revocation
+#
+
+if ! [ "root" = "$(id -u -n)" ]; then
+    echo "ERROR: ${0} must be run as root!"; exit 1
+fi
+
+VPN_URL=http://localhost
+
+[ -f /etc/vpn-provisioning ] && . /etc/vpn-provisioning
+
+printf "DNS name of the intermediate Web Server [%s]: " "${INTERMEDIATE_FQDN}"; read -r USER_INPUT
+INTERMEDIATE_FQDN=$(echo "${USER_INPUT:-${INTERMEDIATE_FQDN}}" | tr '[:upper:]' '[:lower:]')
+
+printf "URL of eduVPN server [%s]: " "${VPN_URL}"; read -r USER_INPUT
+VPN_URL=${USER_INPUT:-${VPN_URL}}
+
+printf "Token of the admin API from eduVPN [%s]: " "${ADMIN_API_TOKEN:0:3}****"; read -r USER_INPUT
+ADMIN_API_TOKEN=${USER_INPUT:-${ADMIN_API_TOKEN}}
+
+printf "Azure Tenant ID [%s]: " "${TENANT_ID}"; read -r USER_INPUT
+TENANT_ID=${USER_INPUT:-${TENANT_ID}}
+
+printf "Application (client) ID [%s]: " "${APPLICATION_ID}"; read -r USER_INPUT
+APPLICATION_ID=${USER_INPUT:-${APPLICATION_ID}}
+
+printf "Secret token of the registered application [%s]: " "${SECRET_TOKEN:0:3}****"; read -r USER_INPUT
+SECRET_TOKEN=${USER_INPUT:-${SECRET_TOKEN}}
+
+###############################################################################
+# SOFTWARE
+###############################################################################
+
+apt update
+apt install -y jq
+
+###############################################################################
+# APACHE
+###############################################################################
+
+cp ./intermediate.example.conf "/etc/apache2/sites-available/${INTERMEDIATE_FQDN}.conf"
+
+mkdir -p "/usr/share/vpn-provisioning/certs"
+cp ./MicrosoftIntuneRootCertificate.cer "/usr/share/vpn-provisioning/certs/MicrosoftIntuneRootCertificate.cer"
+
+mkdir -p "/usr/share/vpn-provisioning/web"
+cp ./index.php "/usr/share/vpn-provisioning/web"
+
+# update hostname
+sed -i "s/vpn.example/${INTERMEDIATE_FQDN}/" "/etc/apache2/sites-available/${INTERMEDIATE_FQDN}.conf"
+sed -i "s/vpn.example/${INTERMEDIATE_FQDN}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update vpn name
+sed -i "s/{vpnUrl}/${VPN_URL//\//\\/}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update tenant id
+sed -i "s/{tenantId}/${TENANT_ID}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update application id
+sed -i "s/{applicationId}/${APPLICATION_ID}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update secret application token
+sed -i "s/{secretToken}/${${SECRET_TOKEN//\\/\\\\}//\//\\/}/" "/usr/share/vpn-provisioning/web/index.php"
```

Comment:
Here is a nested parameter as well, please fix according to https://www.shellcheck.net/wiki/SC2299